.github/workflows/build-validate-publish.yml

```diff
@@ -9,8 +9,14 @@ name: build-validate-publish
 
 on:
   pull_request:
+    paths-ignore: [ "Dockerfile" ]
```

Comment:
Is this just `Dockerfile` at the root of the repository (which doesn't exist) or does it match all files called `Dockerfile`?

Reply:
Totally correct, it used to sit at root but now in subdirectory, made a fix